### ISSUE TITLE
Change the visual display if aria-current="page" is implemented

### ISSUE THREAD

**Issue by bryanjonker-illinois** [web-illinois/ilw-header-menu#31]
If the aria-current is used, there should be a change in how the page looks. Please use the Figma documentation for exact details.

**Comment by bryanjonker-illinois** [comment]
From Bryan:

For example, say I am on the following link: https://builder3.toolkit.illinois.edu/philosophy/index.html

The "Philosophy" page is under "Information". Should aria-current="page" be on both the "Information" main menu item and the "Philosophy" sub-menu item? 

From Keith: 
 
In your example, aria-current="page" should only be on the "Philosophy" sub-menu item and not its parent. To my understanding, the aria-current attribute should only be added to one element in a set of elements. The principle is that there can only be one current page, step, etc. For a navigation menu, an items parent and child cannot both be loaded. This property is about the semantics of the currently active item, not interactivity, so the principle still applies when the parent is not a page. While it is tempting to add it to the parent menu item, the parent is not the current page. As such, it should not get aria-current as well as the child.
